content/news/2023-06-12-call-for-paper-now-open/index.md

```diff
@@ -0,0 +1,64 @@
+
+---
+title: "Call for Paper @UbuConAsia 2023 Now Open! 📢"
+date: 2022-06-12T16:30:01+09:00
+authors:
+    - name: Saputro Aryulianto
+      bio: OpenStack & OpenInfra Indonesia | Local Team, Website, Sponsorship and Finances
+      email: aryuliantos@gmail.com
+      launchpad: aryulianto
+      github: aryulianto
+      profile: https://avatars.githubusercontent.com/u/8587010?v=4
+      linkurl: https://aryulianto.com
+      linklabel: Website
+---
+
+LibreOffice Conf Asia x UbuCon Asia 2023 (hereafter LOUCA23) is an event that brings together Linux and Open Source Software (OSS) activists, contributors, users, communities, and businesses in the Asian region, especially those related to and focused on LibreOffice and Ubuntu projects. This year, LOUCA2023 will be held on 7-8 October 2023 at Universitas Sebelas Maret (UNS), Surakarta, Indonesia!
+
+## We offer two types of presentations:
+* Parallel classes (45 minutes presentation including Q&A)
+* Lightning talks (10 minutes presentation with no Q&A).
+
+## For both events, the speakers can choose the following topics:
+
+Here is some tracks that available to choose for your best topics!
+
+## LibreOffice Track
+* LibreOffice Development
+* LibreOffice Extensions 
+* Development
+* LibreOffice Quality Assurance (QA)
+* LibreOffice l10n
+* LibreOffice Design
+* LibreOffice Community Engagement
+* LibreOffice Tips & Tutorials
+* LibreOffice Documentation
+* LibreOffice Online
+* LibreOffice Support & Ecosystem
+* LibreOffice Interoperability & Open Document Format
+
+##  Ubuntu Track
+* Ubuntu Desktop
+* Ubuntu Cloud and Infrastructure
+* Ubuntu Localizations,Internationalizations, and Accessibility
+* Ubuntu IoT, Embedded, Robotics, Appliances
+* Ubuntu Packaging
+* Ubuntu Documentations, QA and Bug triage
+* Ubuntu Security, Compliance and Kernel
+* Data and AI: Ubuntu for Data Science, Data Engineering, AI and ML workloads
+* Ubuntu Content and Design
+* Ubuntu Community, Diversity, Local Outreach and Social Context
+* Ubuntu Windows Subsystem for Linux(WSL)
```

Comment:
UbuCon tracks have some changes compared to last year. Please apply changes based on this page https://events.canonical.com/event/13/program

Reply:
Done, so I need to update on CFP system too

Reply:
Correct :)